docs/getting_started/prepare_environment.md

```diff
@@ -1,61 +1,39 @@
 # Tutorial: Prepare your Environment
 
-Before starting this tutorial, you will need to setup your development machine.  These instructions were tested on an x86_64 based machine using Ubuntu 20.04.
+Before starting this tutorial, you will need to setup your development machine.  These instructions were tested on an x86_64 based machine using Ubuntu 22.04 and Azure Linux 2.0.
```

Comment:
We use 2.0 here but in `Clone Azure Linux and Build the Toolkit` give `3.0-stable` as an example of a stable tag to use. I think it would be good to consistently use one version of AZL. Would switching to 3.0 in this line be wrong?

Reply:
I've not actually built using a 3.0 image in quite a while, and didn't want to over promise on issues. afaik it works fine but... 2.0 is definitely the most stable option right now.

Reply:
```suggestion
Before starting this tutorial, you will need to setup your development machine.  These instructions were tested on an x86_64 based machine running Ubuntu 22.04 and a machine running Azure Linux 2.0.
```